packages/schemas/carbon-resources.schema.json

```diff
@@ -296,6 +376,26 @@
             "additionalProperties": false,
             "oneOf": [{ "required": ["title", "path"] }, { "required": ["title", "items"] }]
           }
+        },
+        "designKits": {
+          "type": "object",
+          "minProperties": 1,
+          "title": "Carbon Design Kit",
+          "description": "Object containing design kits",
+          "patternProperties": {
+            "^.*$": {
+              "description": "Design kits contain elements, guidelines, user interface components and patterns, and wireframes for various design tools as the design counterpart to coded assets.",
+              "type": "object",
+              "properties": {
+                "$ref": {
```

Comment:
I thought we were going to allow inline declaration as well, are we allowing $ref only?

Reply:
Ah, good call. I had only thought of inline declaration for assets in a library, as part of "phase II". But it makes sense to allow that here too.

I can update this so it's a `$ref` or design kit properties.